### DISCUSSION TITLE
VFO priority, so one will be muted if there's signal in another frequency

### DISCUSSION THREAD

**Discussion by sergeibit** [egzumer/uv-k5-firmware-custom#607] [General]
I'm trying to listen to multiple frequencies by scanning/hopping between them by using the dual VFO.

One of them there's a continuous stream (Airport ATIS), while others have signal only on demand (e.g, by someone pressing PTT).

Is there some kind of "channel priority" or "VFO priority", so "lower priority" channels are muted automatically if there's signal in a "higher priority" channel.

Scenario example: 
VFO 1 is scanning/hopping a list of frequencies.
VFO 2 is demodulating the ATIS channel.
Then, VFO 1 detects an upcoming signal in the Tower channel
The radio then switch to VFO 1 (i.e, ATIS channel will play)
When the VFO 1 signal is gone, it comes back to VFO 2 at the fixed frequency.

Please, understand: the problem is not dual VFO scanning.

The problem I'm trying to solve is stop demodulating a continuous signal in one VFO if a signal is detected in another frequency, and demodulate that frequency instead, even if there's still signal in the another frequency.

It could be useful in a scenario like the one I have, but also for emergency or other important frequency monitoring.

If there's not a way to do it currently, do you think it's something possible to be implemented by software? I mean, the hardware allow something like that?

Thank you very much.

**Comment by AubsUK** [comment]
I don't think the scenario you'd like is possible.  I think the radio can only listen to one frequency at any one time, so if VFO2 is listening to something, the only way for it to periodically check VFO1 for any activity would be to switch frequencies and back.  This would cause breaks in the transmission on VFO2 and would be very annoying.

Even the spectrum analyzer scans through each frequency (albeit very quickly), so it isn't listening to all frequencies at the same time (you can see it scan through when it loads and every now and then when there's noise.
